includes/model/class-user.php

```diff
@@ -150,20 +158,15 @@ public function get_icon() {
 
 	public function get_image() {
 		$header_image = get_user_option( 'activitypub_header_image', $this->_id );
-		$image_url    = null;
-
-		if ( $header_image ) {
```

Comment:
I think we can't remove the `\wp_get_attachment_url( $header_image );` because the wp-admin settings only saves the attachment-id. If the EMA plugin is doing it differently, we have to unify code.

Reply:
I think I found that we were saving just the URL, but I'll circle back and check

Reply:
You were right!